### ISSUE TITLE
Description of instrumentation environment is not able to be clean out.

### ISSUE THREAD

**Issue by youjenli** [inspectIT/inspectIT#335]
If we right click `edit` to modifiy specific environment at  instrumenation manager of InspectIT **1.7.9.92**, then remove all texts inside description field of environment and click `OK` to update description.

![-2017-8-14- -03-20-32](https://user-images.githubusercontent.com/1283378/29262225-0ba1a6b4-8106-11e7-968f-e2cd6fe72c71.png)

Then you will find out that CMR does not clean out description in environment settings.

![-2017-8-14- -03-21-52](https://user-images.githubusercontent.com/1283378/29262232-0f954136-8106-11e7-8f23-0970d560f08d.png)

Is this a bug?

**Comment by ivansenic** [comment]
Yep seams like a bug, we'll open a ticket. Thanks for the report.
